r124/inref/ydb312gtm8182b.m

```diff
@@ -12,34 +12,31 @@
 ;
 
 ydb231gtm8182b
+
 	DO getPaths
 
 	set $ZGBLDIR=INST1gbldir
 	set x=$get(^a)
-	set $ZGBLDIR=INST3gbldir
 
 	set incrtrapNODISP=1
 	set $ZTRAP="goto incrtrap^incrtrap"
-	write "$zrealstor BEFORE $ZPEEK loop is ",$zrealstor,!
+	write "$zrealstor BEFORE $ZPEEK of jnlpool_ctl_struct.ftok_counter_halted loop is ",$zrealstor,!
 	for i=1:1:1000 do zpeek
-	write "$zrealstor AFTER  $ZPEEK loop is ",$zrealstor,!
+	write "$zrealstor AFTER  $ZPEEK of jnlpool_ctl_struct.ftok_counter_halted loop is ",$zrealstor,!
 
 	quit
 
-zpeek ;
-	; Doing a \$zpeek instead of PEEKBYNAME in order to avoid ^incrtrap issues
-	; write \$\$^%PEEKBYNAME("jnlpool_ctl_struct.ftok_counter_halted")
```

Comment:
Please keep this PEEKBYNAME line commented as it is useful to know what the $ZPEEK command is equivalent to.